docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -647,6 +647,18 @@ and returns no response.
 await liveblocks.deleteRoom("my-room-id");
 ```
 
+#### Liveblocks.prewarmRoom [#get-rooms-roomId-prewarm]
+
+Prepares a room for connectivity, making the eventual connection faster. Use
+this when you know you'll be loading a room but are not yet connected to it.
```

Comment:
Should we say "use this when you know you'll be connecting to a room soon" or something? I think the difference between "loading" and "connecting" can be a bit unclear to end users.

Reply:
Should we also mention "soon" technically means within 10 seconds? If you prewarm the room and wait 10 seconds, the prewarm will be discarded due to hibernation.